pkg/frontend/load.go

```diff
@@ -482,7 +482,7 @@ func isWriteBatchTimeoutError(err error) bool {
 	return errors.Is(err, context.DeadlineExceeded)
 }
 
-func judgeInterge(field string) bool {
+func judgeInteger(field string) bool {
```

Comment:
This function need a comment.   What are you judging?

Reply:
ok

Reply:
Did you?   How can I see the diff ...